Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingBifurcatedImageBuffer.h

```diff
@@ -23,29 +23,38 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "config.h"
-#include "DynamicContentScalingDisplayList.h"
-
-#include "SharedBufferReference.h"
+#pragma once
 
 #if ENABLE(RE_DYNAMIC_CONTENT_SCALING)
 
+#include "DynamicContentScalingImageBufferBackend.h"
+#include "ImageBufferBackendHandleSharing.h"
+#include <WebCore/BifurcatedGraphicsContext.h>
+#include <WebCore/DynamicContentScalingDisplayList.h>
+#include <WebCore/ImageBuffer.h>
+#include <WebCore/ImageBufferCGBackend.h>
+#include <wtf/IsoMalloc.h>
```

Comment:
Do we really need to include these header files? Can't we replace some of them by forward declarations?

Reply:
drastically reduced